packages/core.rb

```diff
@@ -79,6 +81,7 @@ class Core < Package
   depends_on 'uchardet'
   depends_on 'unzip'
   depends_on 'xzutils'
+  depends_on 'xxhash'
```

Comment:
We're supposed to be reducing the core dependencies. :)

Reply:
Yes but that's an rsync dep!